src/main/java/com/controller/QuoteController.java

```diff
@@ -37,22 +36,38 @@ public ResponseEntity createQuote(@RequestBody Quote quote) {
      * Get all quotes.
      */
     @GetMapping(path = "/quotes", produces = "application/json")
-    public ResponseEntity<List<Quote>> getQuotes() {
+    public ResponseEntity<List<Quote>> getQuotes(@RequestParam(required = false) String author,
+                                                 @RequestParam(required = false) Integer rating) {
+
+        if (author != null && rating != null) {
+            List<Quote> allQuotesOfAuthorWithRating = quoteService.getAllQuotesOfAuthorWithRating(author,rating);
+            return new ResponseEntity<>(allQuotesOfAuthorWithRating, HttpStatus.OK);
```

Comment:
`return new ResponseEntity<>(allQuotesOfAuthorWithRating, HttpStatus.OK);` is same in all cases. Extract it.